.github/workflows/ci.yml

```
@@ -91,20 +310,139 @@ jobs:
 
       - name: Run E2E tests
         run: pnpm test:e2e:ci
+        continue-on-error: true
+        id: e2e_tests
+
+      - name: Parse E2E test results
+        if: always()
+        run: |
```

actionlint (1.7.7)

got unexpected character '\\' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '

(expression)